keyboards/sofle_pico/readme.md

```diff
@@ -0,0 +1,43 @@
+# Sofle Pico Keyboard
+
+![Sofle Pico](https://raw.githubusercontent.com/JellyTitan/Sofle-Pico/main/docs/images/qmk/sofle_pico_hero_image.jpeg)
+![Sofle Pico PCB](https://raw.githubusercontent.com/JellyTitan/Sofle-Pico/main/docs/images/qmk/sofle_pico_pcb_hero_image.jpeg)
+Sofle Pico is a 60% Raspberry Pi Pico powered split keyboard boasting a 4x6 layout with an aggressive columnar stagger. It has 5 "thumb" keys on either side, support for 2-4 rotary encoders, per-key RGB lighting, 128x64 OLEDs and a tenting puck mount.
+
+- Keyboard Maintainer: [Ryan Neff](https://github.com/JellyTitan)
+- Hardware Supported: Sofle Pico PCB v3.5.x, Raspberry Pi Pico
+- Hardware Availability: [PCB, Case, Parts List](https://github.com/JellyTitan/Sofle-Pico)
+
+Remember you have to flash both halves of the keyboard for it to work!
+
+Make example for this keyboard (after setting up your build environment):
+
+    make sofle_pico:via
+
+Flashing example for this keyboard:
+
+```bash
+# For flashing the left half...
+make sofle_pico:default:uf2-split-left
+# or the qmk cli equivalent:
+qmk flash -kb sofle_pico --keymap default -bl uf2-split-left
+
+# For flashing the right half...
+make sofle_pico:default:uf2-split-right
+# or the qmk cli equivalent:
+qmk flash -kb sofle_pico --keymap default -bl uf2-split-right
+```
```

Comment:
```suggestion
    make sofle_pico:default

Flashing example for this keyboard:

    make sofle_pico:default:flash
```

Reply:
I added the changes you mentioned above. Since handedness is set by `ee_hands`, I added a note about the initial EEPROM flash. 

I couldn't find anything in the QMK docs about how this should be structured in the README, so I cribbed the example I found in the [Splitty Readme](https://github.com/qmk/qmk_firmware/blob/master/keyboards/splitty/readme.md)
![image](https://github.com/qmk/qmk_firmware/assets/72168556/25286914-84bc-4c72-8627-f7d5bcfd9eaa)


Is there a better way to approach this?

Reply:
It's moreso the syntax/usage of `make` consistency across readme's. Something like this would be fine

```
Make example for this keyboard (after setting up your build environment):

    make sofle_pico:default

Flashing example for this keyboard:

    make sofle_pico:default:uf2-split-left
    make sofle_pico:default:uf2-split-right

```